Makefile

```diff
@@ -6,7 +6,21 @@ endif
 ifndef VENV_NAME
 	VENV_NAME = locust-venv
 endif
-VENV_DIR = $(HOME)/.pyenv/versions/$(PY_VERSION)/envs/$(VENV_NAME)
+ifndef USE_ASDF
+		USE_ASDF:=false
+endif
+
+# ASDF versions
+# global $HOME/.tool-versions
+# local $PWD/.tool-versions
+# $(HOME)/.asdf/installs/python/$(PY_VERSION)
+# PYENV versions
+# $(HOME)/.pyenv/versions/$(PY_VERSION)
```

Comment:
What does this commented code mean?

Reply:
@sandy-wright  I was working out where things were stored, probably not needed here.